docs/userguide/example-uses.rst

```diff
@@ -0,0 +1,195 @@
+                        light: 97.0
+                        light_to_moderate: 99.1
+                        moderate: 99.6
+                        moderate_to_severe: 99.8
+                        severe: 99.9
+                phases:
+                    - thresholds
+                    - histogram
+            evaluation_phases:
+                phases:
+                    - probabilities
+                    - edr
+                evaluation_config:
+                    evaluation_data_dir: met_data/era5/evaluation_data
+
+This configuration can be launched using the command below,
+
+.. code-block::
+
+    $ rojak run turbulence-probability-config.yaml
+
+To monitor the progress of the process through the Dask :doc:`dask:dashboard`, go to `http://localhost:8787/status <http://localhost:8787/status>`_.
+
+.. note::
+
+    This configuration was run on the HPC with 920GB of memory.
+    It is possible that it does not require as much. It is likely to require a minimum of 42GB of memory.
+    Moreover, as the default behaviour of dask (see :doc:`dask:configuration`) is to write to ``/tmp`` its ability to spill to disk may be limited to the amount of memory in your system.
+
+
+EDR Snapshot
+--------------------------------------
+
+By default, ``rojak`` does not produce snapshot plots. This can be achieved through a few different ways.
+The method shown below is the least involved, thus has the least room for customisation.
+
+.. _edr-snapshot-fig:
+
+.. figure:: _static/multi_edr_f3d_ti1.png
+    :align: center
+
+    6-hour forecast of eddy dissipation rate (EDR) at 200 hPa for the three-dimensional frontogenesis (F3D) and turbulence index 1 (TI1) on the 1st of December 2024 at 00:00
+
+Instead of performing the analysis globally like in the :ref:`previous section <computing-turbulence-percentage>`,
+the domain can be limited (like in :numref:`edr-snapshot-fig`) by specifying it in like in lines 2-6 in :numref:`edr-snapshot-config-yaml`.
+
+.. code-block:: yaml
+    :linenos:
+    :caption: edr-snapshot-config.yaml
+    :name: edr-snapshot-config-yaml
+
+    data_config:
+        spatial_domain:
+            maximum_latitude: 70
+            maximum_longitude: 60
+            minimum_latitude: 0
+            minimum_longitude: -150
+    image_format: png
+    name: eighties
+    output_dir: output
+    plots_dir: plots
+    turbulence_config:
+        chunks:
+            pressure_level: 3
+            latitude: 721
+            longitude: 1440
+        diagnostics:
+            - f3d
+            - ti1
+        phases:
+            calibration_phases:
+                calibration_config:
+                    calibration_data_dir: met_data/era5/calibration_data
+                phases:
+                    - histogram
+            evaluation_phases:
+                phases:
+                    - edr
+                evaluation_config:
+                    evaluation_data_dir: met_data/era5/evaluation_data
```

Comment:
As above, I needed to update to `evaluation_data_dir: met_data/met_data/era5/evaluation_data` based on how the files were downloaded from CDR.